drivers/staging/vc04_services/bcm2835-codec/bcm2835-v4l2-codec.c

```diff
@@ -79,6 +79,10 @@ static bool advanced = true;
 module_param(advanced, bool, 0644);
 MODULE_PARM_DESC(advanced, "Use advanced deinterlace");
 
+static int field;
```

Comment:
Too generic a name to be a module parameter, and likely to get confused with a local var called ```field```.

```field_override```?